templates/internet-of-things/appstore.html

```diff
@@ -10,237 +10,302 @@
       <ul class="p-list">
         <li class="p-list__item is-ticked">Pin to a specific version of software on your devices</li>
-        <li class="p-list__item is-ticked">Determine which devices connect to your proxy</li>
+        <li class="p-list__item is-ticked">Determine which devices connect to your Enterprise App Store</li>
         <li class="p-list__item is-ticked">Manage users allowed to override revisions</li>
         <li class="p-list__item is-ticked">Maximise bandwidth and speed by caching downloads</li>
       </ul>
+      <a class="p-button--positive u-no-margin--bottom" href="https://docs.ubuntu.com/snap-store-proxy/en/"><span class="p-link--external">Snap store proxy</span></a>
     </div>
   </div>
 </section>
 
-<section class="p-strip--accent is-dark">
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2>Dedicated App Store</h2>
+    <h3 class="p-heading--four">Curate your own app collection with custom pricing and unique applications.</h3>
+    <p>Optional managed app store includes role-based access controls, complete control of application versions, updates and controlled rollouts for $15,000 per year. We enable integrators and VARs to operate app stores on behalf of their clients. Access to the full Ubuntu app ecosystem requires device certification.</p>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <h2>Enterprise App Store</h2>
+    <h3 class="p-heading--four">Control snap distribution inside your firewall, and cache snap traffic for network efficiency.</h3>
+    <p>The enterprise snap store enables your administrators to control snap updates and upgrades. Ensure that all device traffic goes through an audited communications channel and determine the precise versions of snaps used inside the business.</p>
+  </div>
+</section>
+
+<section class="p-strip--light">
   <div class="row u-equal-height">
-    <div class="col-8">
-      <p class="p-heading--three">If you’re looking for ways to distribute and manage software on your IoT devices, sign up for a free consultation.</p>
+    <div class="col-7">
+      <h2>Ready to innovate? <br class="u-hide--small" />We would love to hear from you!</h2>
+      <p class="p-heading--four">Our store team would be happy to hear more about your use cases and share advice. Do not hesitate to contact us.</p>
       <p style="margin-top: 2rem;">
-        <a class="p-button--positive js-invoke-modal" href="/internet-of-things/contact-us?product=appstore" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact us', 'eventLabel' : 'Telcos - Top CTA' });">Contact us</a>
+        <a class="p-button--positive js-invoke-modal" href="/internet-of-things/contact-us?product=appstore" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact us' });">Contact us</a>
```

Comment:
```suggestion
        <a class="p-button--positive js-invoke-modal" href="/internet-of-things/contact-us?product=appstore" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact us' });">Get in touch</a>
```

Reply:
This looks good but I do not have permission to push to this branch. Help to land the fix in a follow up?